### DISCUSSION TITLE
Usage of compute credits

### DISCUSSION THREAD

**Discussion by ghost** [cirruslabs/cirrus-ci-docs#1206] [Q&A]
My build takes more than 2 hours, which is not possible with free account, due to which I need to buy compute credits. And if my build takes extra 30 minutes, and if I trigger at least 7-8 builds within a week and if I buy 2 compute credits, then with how much credits will I be left at the end of the month ?

**Comment by fkorotkov** [comment]
In this scenario your task is taking 150 minutes. If you are using the max available CPUs which is 8 then task will consume `150 * 8 = 1200` CPU minutes. With 8 builds per week you'll consume `9600` CPU minutes which with the price of 4 credits per 1000 minutes will result in `38.4` compute credits consumed weekly.

**Reply by ghost** [reply]
So how much will 38.4 compute credits cost in Indian Rupees?

**Reply by fkorotkov** [reply]
One credit costs one United States Dollar. At the moment of writing this comment `38.4 USD` is equal to `3,161.93 INR`.
